trapezoidal.go

```diff
@@ -0,0 +1,32 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package integrate
+
+import "sort"
+
+// Trapezoidal returns the integral value of the function sampled by the set of points (x,y).
```

Comment:
My wording:

`Trapezoidal estimates the integral of a function from a set of evaluations of the function using the trapezoidal rule. It estimates
  \int_a^b f(x) dx
where f[i] is the value of the function at x[i]. The trapezoidal rule makes piecewise trapezoidal approximations to the function, and estimates 
  \int_x[i]^x[i+1] f(x)dx
as 
  (x[i+1] - x[i]) * (f[i] + f[i+1])/2
More details on the trapezoidal rule can be found at https://en.wikipedia.org/wiki/Trapezoidal_rule

The x and y input data points must be sorted and of equal length (>1).`